src/client/Client.h

```diff
@@ -854,7 +854,7 @@ class Client : public Dispatcher, public md_config_obs_t {
    * Resolve file descriptor, or return NULL.
    */
   Fh *get_filehandle(int fd) {
-    ceph::unordered_map<int, Fh*>::iterator p = fd_map.find(fd);
+    auto p = fd_map.find(fd);
```

Comment:
If you're gonna fix this, please also update the name to `it`. (`p` is short for `std::pair` but this is an iterator!)

Reply:
Okay, will fix it.